Instant-Messenger/src/server/server.cpp

```diff
@@ -312,4 +359,199 @@ namespace server {
         return socketId;
     }
 
+
+    void Server::connectToRmServers() {
+        for(int connectMachineNumber = rmNumber; connectMachineNumber < MAX_RM; connectMachineNumber++) {
+            int *connectSocket = (int *) calloc(1, sizeof(int *));
+            char ip_address[10] = "127.0.0.1";
+            struct sockaddr_in rm_connect_socket_addr{};
+
+            rm_connect_socket_addr.sin_family = AF_INET;
+            rm_connect_socket_addr.sin_port = htons(RM_BASE_PORT_NUMBER + connectMachineNumber + 1); // port is base port + next machine RM number (rmNumber+1)
```

Comment:
Check this out to maybe flexibilize it

Reply:
Already discussed with Gabriel. Lets first make everything work and then we can tackle enhancements and refactorings.